benchmark/clBench/clblast/blas2/spr2.cpp

```diff
@@ -0,0 +1,162 @@
+/**************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename spr2.cpp
+ *
+ **************************************************************************/
+
+#include "../utils.hpp"
+
+template <typename scalar_t>
+std::string get_name(char uplo, int size, scalar_t alpha, int incX) {
```

Comment:
`incY` is not passed as a benchmark parameter and it is set to 1. Adding it here, I think it would make it redundant, what do you think?
We could change the benchmark to pass it as parameter and update also `cuBLAS` and `rocBLAS` benchmarks.

Reply:
It is the same for `incX` as that is also set to `1`. I think it would be good to have `incY` here as a parameter for completeness sake. We can address this change in the `spr2` benchmarks from other `libraries` as well.